test_ttt_game.py

```diff
@@ -9,7 +9,7 @@ def test_01_horizontal_three_of_a_kind_identifies_three_of_a_kind():
     assert g.horizontal_three_of_a_kind('X')
 
 
-def test_02_horizontal_three_of_a_kind_identifies_three_of_a_kind():
+def test_02_horizontal_three_of_a_kind_identifies_three_of_a_kind_twice():
     g = Game()
```

Comment:
Can I suggest a new name for horizontal_three_of_a_kind ?
Something along the lines of 
`isRowComplete`

Reply:
Roger that in the next iteration.